test/Escrow.t.sol

```diff
@@ -227,8 +226,6 @@ contract EscrowTest is AtlasBaseTest {
             .build();
         defaultAtlasWithCallConfig(callConfig);
 
-        vm.expectEmit(false, false, false, true, address(atlas));
-        emit MEVPaymentFailure(address(dAppControl), callConfig.encodeCallConfig(), address(0), defaultBidAmount);
         this.executeHookCase(false, 1, noError);
     }
```

Comment:
This test can be removed if no more event to test.

Reply:
Removed, thanks